Aptoide/aptoide-client-v8
# [feature/ASV-1209] Remove ad ab test

**What does this PR do?**

   Removes appview interstitial AB testing from the cobrand flavour, so that interstitials may only show in vanilla flavour.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] IronSourceAdExperiment.java

**How should this be manually tested?**

 Check if in cobrand no appview interstitials are being shown and no ab test is performed. Also check if the vanilla flavour still has the appview interstitials and the ab test is working

**What are the relevant tickets?**

  [ASV-1209](https://aptoide.atlassian.net/browse/ASV-1209)

**Code Review Checklist**
